layouts/partials/posts/commento.html

```diff
@@ -1,4 +1,4 @@
-{{- if and (not (eq .Site.Params.commentoUrl "" )) (eq (.Params.disable_comments | default false) false) -}}
+{{- if and (isset .Site.Params "commentourl") (not (eq .Site.Params.commentoUrl "" )) (eq (.Params.disable_comments | default false) false) -}}
```

Comment:
Maybe we should make the same change in `disqus.html` too.

Reply:
or just use Hugos [internal diqus template](https://gohugo.io/templates/internal/#disqus):

```
{{ template "_internal/disqus.html" . }}
```

Reply:
Yeah, we use it (here)[https://github.com/luizdepra/hugo-coder/blob/master/layouts/partials/posts/disqus.html].

Reply:
@alrayyes We all make mistakes, it's not a big deal :)

@luizdepra I made the same change to disqus.html, but I don't see a similar change in output?  Possibly I'm just not triggering it?

Reply:
Yep. Just to be sure.